lib/views.js

```diff
@@ -89,60 +25,36 @@ function configure(app) {
   }
   const paths = viewPaths.map(function(p) {return path.resolve(p);});
 
-  // add swig as the default template engine
-  app.engine('html', function(path, options, callback) {
-    let view;
-    try {
-      const swig = new swigcore.Swig({
-        autoescape: false,
-        cache: bedrock.config.views.cache ? 'memory' : false,
-        loader: swigLoaders.multipath({base: paths})
-      });
-      view = swig.compileFile(path)(options);
-    } catch(err) {
-      return callback(err);
-    }
-    callback(null, view);
-  });
+  // add the default template engine
+  // TODO: use configuration to allow other engines
```

Comment:
Looks like config is being used now, is that right?

Reply:
Yup. Forgot to remove comment.